src/main/scala/com.snowplowanalytics/weather/providers/openweather/Api.scala

```diff
@@ -13,14 +13,10 @@
 package com.snowplowanalytics.weather
 package providers.openweather
 
-/**
- * Module for various predefined values in OWM specification
- */
+/** Module for various predefined values in OWM specification */
 object Api {
 
-  /**
-   * Enums for measurements for history requests
-   */
+  /** Enums for measurements for history requests */
```

Comment:
It would be nice to switch this to sealed hierarchy, even thought that would require a change in SCE.